### PR TITLE
chore: add announcement bar

### LINKED ISSUE #131: Announcement bar with link to latest major version release notes
**What would you like to be able to do?**

Users of Renovate frequently get confused and open issues/discussions to report "bugs" that are actually intended behavior from a breaking `major` release of Renovate.

I think we can reduce instances of readers getting confused by mentioning on our Docs homepage what our latest `major` release is, and to give them a link to read the release notes.

**Did you already have any implementation ideas?**

Material for MkDocs allows you to override [^overriding-blocks] the defaults with a custom `overrides` folder, where you can put a `main.html` file where you use a templating language to add your own custom HTML. 

So we can use these features to add a announcement bar. [^announcement-bar]

Unfortunately, we cannot let our readers "mark the announcement bar as read" [^mark-as-read] as that's a feature for people who sponsor Material for MkDocs, which costs $10 a month.
This means that the announcement bar stays visible when you visit a anchor link that takes you to the top of _any_ page. The announcement bar will disappear when scrolling down though.

**Proof of concept:**

Proof of concept branch: https://github.com/HonkingGoose/renovatebot.github.io/tree/announcement-bar-poc

**Screenshot:**

announcement-bar-proof-of-concept

[^announcement-bar]: https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-header/#announcement-bar

[^mark-as-read]: https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-header/#mark-as-read

[^extending]: https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme

[^overriding-blocks]: https://squidfunk.github.io/mkdocs-material/customization/#overriding-partials

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Add announcement bar with a link to the latest Renovate `major` version

## Context:

Closes #131.

### CODE REVIEW

File: overrides/main.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!-- This file has the content for our announcement bar. -->
+<!-- Use target="_blank" when linking to a external site, so the link opens in a new tab. -->
+<!-- https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-header/#announcement-bar -->
+<!-- https://squidfunk.github.io/mkdocs-material/customization/#overriding-blocks -->
+
+{% extends "base.html" %}
+
+{% block announce %}
+Read the release notes for our recent major release: <a href="https://github.com/renovatebot/renovate/releases/tag/32.0.0" target="_blank">32.0.0</a>.
```
Comment (rarkins) [inline overrides/main.html:9]: In this case it was pretty long ago, and I don't think we need to show this to everyone forever. Maybe start with v33 and leave it up for a month?

Comment (HonkingGoose) [reply overrides/main.html:9]: > In this case it was pretty long ago, and I don't think we need to show this to everyone forever.

Fair enough. I just wanted to put something in there to test it actually works in the server preview.

> Maybe start with v33 and leave it up for a month?

That's an option, or we can:

1. Nudge our adventurous users to try out `configMigration` so we can get more bugs fixed before general release.
2. Highlight the new Swissquote use case article.

Can you tell me what you want me to do? 😉

Comment (rarkins) [reply overrides/main.html:9]: Pointing to config migration sounds like a good idea, especially once it's a bit more stable. I think we still have closing/reopening problems related to cache though.

Comment (HonkingGoose) [reply overrides/main.html:9]: How about this sequence of announcements:

1. Highlight the Swissquote article first
2. Then once it's stable enough, nudge users to try `configMigration` before general release
3. Highlight any major Renovate version changelog, so users read the breaking changes

```suggestion
Read how Swissquote uses Renovate in the new <a href="https://docs.renovatebot.com/user-stories/swissquote/" target="_blank">Swissquote user story</a> on our site.
```

**Review by rarkins — APPROVED** [review]

**Pull request merged**